files/zh-cn/learn/css/css_layout/grids/index.html

```diff
@@ -83,6 +83,7 @@ <h3 id="定义一个网格">定义一个网格</h3>
 
 <p>在规则里加入你的第二个声明。刷新页面后，你会看到子项们排进了新定义的网格中。</p>
 
+<div id="Grid_1">
```

Comment:
Please make sure for every added `<div id="Grid_*"` has it's own close tag.
This article a bit difference with others cause it do not wrap code in a `pre` tag instead a hidden `div`.

Reply:
You're right. Thanks for catching this!